src/pybind/mgr/dashboard/frontend/src/app/shared/services/auth-storage.service.spec.ts

```diff
@@ -34,7 +34,7 @@ describe('AuthStorageService', () => {
   });
 
   it('should be SSO', () => {
-    service.set(username, '', {}, true);
+    service.set(username, {}, true);
```

Comment:
Shouldn't the similar change applied here too ?
 it('should not be SSO', () => {
  ----->  service.set(username, '');	    service.set(username);

Reply:
Yes, fixed this. thanks @avanthakkar :+1: